Components/Combat/DefaultLootProvider.cs

```diff
@@ -529,7 +529,7 @@ public bool ShouldStash(ACDItem item)
             // Keep any high gems placed in backpack while levelling, so we can socket items with them.
             if (item.IsGem &&
                 item.Stats.GemQuality >= GemQuality.Marquise &&
-                ZetaDia.Me.Level < 70)
+                ZetaDia.Me?.Level < 70)

```

Comment:
Would be safer to consider Me.Level above 70 by default here. 

something like  ZetaDia.Me?.Level ?? int.MaxValue < 70